module2/src/com/epam/arraysofarrays/MatrixWorker.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.arraysofarrays;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+
+public class MatrixWorker {
+    //task 1
+    public static boolean findOddColumnsWhereFirstElementGreaterThanLast(int[][]matrix) {
+        for (int j = 0; j < 5; j++) {
+            for (int i = 0; i < 5; i += 2) {
+                if (matrix[0][i] > matrix[4][i]) {
+                    System.out.print(matrix[j][i] + "\t");
+                }
+            }
+            System.out.println();
+        }
+        return true;
+    }
+    //task 2
+    public static boolean findElementsTheMainDiagonal(int[][] matrix) {
+        System.out.print("Elements standing on the diagonal = ");
+        for( int i = 0; i < 5; i++) {
+            System.out.print(matrix[i][i] + "\t");
+        }
+        return true;
+    }
+    //task 3
+    public static boolean outputGivenRowAndColumn(int[][] matrix, int row, int column) {
```

Comment:
The same comments as in the 1st task + it is better to use word - "print" than "output"
```suggestion
    public static boolean printRowAndColumn(int[][] matrix, int row, int column) {
```